packages/@aws-cdk/aws-backup/lib/vault.ts

```diff
@@ -0,0 +1,202 @@
+   * The name to assign the vault.
+   * The name is case-sensitive and must be between 2 and 50 characters or hyphens.
+   * @see https://docs.aws.amazon.com/aws-backup/latest/devguide/creating-a-vault.html
+   */
+  readonly backupVaultName: string;
+  /**
+   * Optional KMS Key to encrypt the backups.
+   * @see https://docs.aws.amazon.com/aws-backup/latest/devguide/creating-a-vault.html
+   * @default Backup Service will create one with the alias `aws/backup`.
+   */
+  readonly encryptionKey?: kms.IKey;
+  /**
+   * Notification configuration based on SNS Topics.
+   * @default configures no notifications
+   */
+  readonly notifications?: VaultNotifications;
+  /**
+   * Optional access policy for the vault.
+   * @see https://docs.aws.amazon.com/aws-backup/latest/devguide/creating-a-vault-access-policy.html
+   * @default Has not access policy.
+   */
+  readonly accessPolicy?: iam.PolicyDocument;
+}
+
+/**
+ * Represents the attributes from a backup vault.
+ */
+export interface BackupVaultAttributes {
+  /**
+   * An ARN string representations.
+   */
+  readonly backupVaultArn: string;
+  /**
+   * A string representation of the backup vault's name.
+   */
+  readonly backupVaultName: string;
+}
+
+/**
+ * Backup Vault where all backups will be stored.
+ */
+export class BackupVault extends BackupVaultBase {
+
+  /**
+   * Import a backup Vault.
+   */
+  public static fromBackupVaultAttributes(scope: Construct, id: string, attrs: BackupVaultAttributes): IBackupVault {
+    class Import extends BackupVaultBase {
+      public readonly backupVaultArn = attrs.backupVaultArn;
+      public readonly backupVaultName = attrs.backupVaultName;
+      public readonly accessPolicy?: iam.PolicyDocument; // dropped
+      public readonly encryptionKey?: kms.IKey; // dropped
+    }
+
+    return new Import(scope, id);
+  }
+
+  public readonly backupVaultName: string;
+  public readonly backupVaultArn: string;
+  public readonly encryptionKey?: kms.IKey;
+  public readonly accessPolicy?: iam.PolicyDocument;
+
+  constructor(scope: Construct, id: string, props: BackupVaultProps) {
+    super(scope, id, {
+      physicalName: props.backupVaultName
+    });
+
+    const accessPolicy = props.accessPolicy;
+    const encryptionKeyArn = props.encryptionKey ? props.encryptionKey.keyArn : undefined;
+    const notifications = this.renderNotifications(props.notifications);
+
+    const resource = new CfnBackupVault(this, 'Resource', {
+      backupVaultName: this.physicalName,
+      encryptionKeyArn,
+      accessPolicy,
+      notifications,
+    });
+
+    this.backupVaultName = this.getResourceNameAttribute(resource.ref);
+    this.backupVaultArn = this.getResourceArnAttribute(resource.attrBackupVaultArn, {
+      service: 'backup',
+      resource: 'backup-vault',
+      resourceName: this.backupVaultName,
+    });
+  }
+
+  private renderNotifications(notifications?: VaultNotifications): CfnBackupVault.NotificationObjectTypeProperty | undefined {
```

Comment:
You need to `topic.grantPublish(new iam.ServicePrincipal('backup.amazonaws.com'))` somewhere